packages/shared/src/coh/helpers.ts

```diff
@@ -0,0 +1,110 @@
+import { LaddersDataArrayObject, LaddersDataObject } from "./types";
+import { levels } from "./coh2-api";
+
+const sortArrayOfObjectsByTheirPropertyValue = (
+  mapsData: Array<Record<string, string>>,
+): Array<Record<string, string>> => {
+  return mapsData.sort((a, b) => {
+    if (a.value < b.value) {
+      return -1;
+    }
+    if (a.value > b.value) {
+      return 1;
+    }
+    return 0;
+  });
+};
+
+const getExportedIconPath = (name: string): string => {
+  return `/resources/exportedIcons/${name}.png`;
```

Comment:
I would probably delete this function from the shared  folder. I believe this function can work properly only in the web package - as it's tied to the resources path  - or can it work in app too?